source/core/query-plans.txt

```diff
@@ -228,7 +228,7 @@ If one of the indexes were dropped, or if a new index ``{ x: 1, a: 1
 change.
 
 Availability
-~~~~~~~~~~~~
+''''''''''''''
```

Comment:
This should be h2 (~) because it applies to both planCacheShapeHash and planCacheKey. With the current presentation as a subsection of planCacheKey, it looks like it only applies to planCacheKey.